docs/src/components/usage.tsx

```diff
@@ -1,27 +1,84 @@
 import React from 'react'
 
-import { A } from './mdx/links'
+import Button from '@pluralsight/ps-design-system-button'
+import Icon, {
+  SocialGithubIcon,
+  ExternalLinkIcon,
+  ListIcon
+} from '@pluralsight/ps-design-system-icon'
 import * as styles from './usage.module.css'
 
 interface UsageProps {
+  figmaUrl: string
   install: string
   import: string
   packageName: string
   version: string
 }
 export const Usage: React.FC<UsageProps> = props => {
   return (
-    <ul className={styles.usage}>
-      <Item label="Version">
-        <A
-          href={`https://github.com/pluralsight/design-system/blob/master/packages/${props.packageName}/CHANGELOG.md`}
-        >
-          {props.version || 'CHANGELOG'}
-        </A>
-      </Item>
-      <Item label="Install">{props.install.trim()}</Item>
-      <Item label="Import">{props.import.trim()}</Item>
-    </ul>
+    <>
```

Comment:
Could remove this fragment tag.

Reply:
I get an error when I do that. Should I leave the fragment tag or do something else? 
![CleanShot 2021-08-31 at 15 49 04](https://user-images.githubusercontent.com/48542/131580303-598bdc85-bb05-4213-8ee8-2ee3e1d9ac4a.png)

Reply:
Oh, sorry, yeah, leave it. I didn't see the second `<ul/>`.